po/ja.po

```diff
@@ -1373,36 +1377,43 @@ msgid "**In-Person Event Checklist (Carpentries Workshops, CarpentryCon, Carpent
 
 #: topic_folders/policies/incident-response.md:36
 msgid "**Online Events and Communications Channels Checklist (Teaching Demonstrations, Community Discussions, Carpentries Instructor Training, Slack Channels, TopicBox)**\n"
 "* Inform the event host/meeting facilitator that there was an issue and send a report via the [incident report form][reporting-form]. \n"
 "* If the incident involves the event host/meeting facilitator, please complete the [incident report form][reporting-form] and rest assured that confidentiality and your experience in our community is our first priority."
-msgstr ""
+msgstr "**オンラインイベントとコミュニケーションチャネルのチェックリスト (ティーチングデモ、コミュニティの議論、カーペントリーズのインストラクターのトレーニング、Slackチャンネル、TopicBox)**\n"
+"* イベントの主催者/会議の進行役に問題があったことを通知し、[インシデントレポートフォーム][reporting-form]を介してレポートを送信します。\n"
+"* インシデントがイベント主催者/会議の進行役に関係している場合は、[インシデントレポートフォーム][reporting-form]に記入してください。機密保持とコミュニティでの経験が最優先事項ですのでご安心ください。\n"
 
 #: topic_folders/policies/incident-response.md:40
 msgid "If a community member has violated the CoC via an online event, the CoC committee can enact a short-term [Termed Suspension](termed-suspension.md), and the reportee’s privileges to all Carpentries communication channels could be suspended until the Code of Conduct Committee has concluded their investigation of the reported incident."
-msgstr ""
+msgstr "コミュニティメンバーがオンラインイベントを介して行動規範に違反した場合、行動規範委員会は短期の一時停止を制定でき、行動規範委員会がレポートされた調査を終了するまで、すべてのカーペントリーズのコミュニケーションチャネルに対するレポートされた人の権限を一時的に止めることができます。"
 
 #: topic_folders/policies/incident-response.md:42
 msgid "Individuals reported often get upset, defensive, or deny the report. Allow them to give any additional details about the incident. However, remember:"
-msgstr ""
+msgstr "レポートされた人は、しばしば動揺したり、自己防衛したり、報告を否定したりします。その人にインシデントに関する追加の詳細を提供することを許してあげてください。ただし、次の点に注意してください:"
 
 #: topic_folders/policies/incident-response.md:44
 # unordered list
 msgid "- It does not matter if they did not intend to hurt anyone; their behaviour still impacted participants negatively."
-msgstr ""
+msgstr "- 誰かを傷つけるつもりがなかったかどうかは関係ありません; その行動はやはり参加者に悪影響を及ぼしました。"
 
 #: topic_folders/policies/incident-response.md:45
 # unordered list
 msgid "- It is not your job to reassure or forgive them."
-msgstr ""
+msgstr "- レポートされた人を安心させたり許したりするのはあなたの仕事ではありません。"
 
 #: topic_folders/policies/incident-response.md:46
 # unordered list
 msgid "- Do not allow the reported person to make an apology to the reporter or impacted person. Often an apology centers the reported person’s feelings and not the person who was impacted. You may accept their apology and offer to pass it on, but you’re not required to if you think it would negatively impact the reporter."
-msgstr ""
+msgstr "- レポートされた人がレポーターまたは影響を受けた人に謝罪することを許可しないでください。多くの場合、謝罪は、影響を受けた人ではなく、レポートされた人の感情を中心にしています。あなたはその謝罪を受け入れてそれを伝えることを申し出るかもしれませんが、それがレポーターに悪影響を与えると思うならあなたはそうする必要はありません。"
```

Comment:
Tried to simplify the part about "you may ... offer to pass it on" -> "you may pass it on"
```suggestion
msgstr "- 報告された人がレポーターまたは影響を受けた人に謝罪することを許可しないでください。多くの場合、謝罪は、影響を受けた人ではなく、報告された人の感情を中心にしています。あなたはその謝罪を受け入れてそれを伝えても構いませんが、それがレポーター（被害者）に悪影響を与えると思うならあなたはそうする必要はありません。"
```